output.tf

```diff
@@ -1,11 +1,7 @@
-output "security_group_id" {
-  value = aws_security_group.elasticache.id
-}
-
-output "endpoints" {
-  value = aws_elasticache_cluster.default.cache_nodes.*.address
+output "primary_endpoints" {
```

Comment:
Why change the name? Think just keeping it `endpoints` will be better backwards compatible.